lib/cdo/shared_constants/pd/teacher2122_application_constants.rb

```diff
@@ -56,21 +56,19 @@ def self.clean_multiline(string)
       },
       choose_your_program: {
         program: clean_multiline(
-          "Which professional learning program would you like to join for the #{SHORT_YEAR}
+          "Which professional learning program would you like to participate in for the #{YEAR}
           school year? Note: this application is only for Computer Science Discoveries and
           Computer Science Principles. If you are interested in teaching Advanced
-          Placement CS A (in Java), visit this
-          [AP CS A overview](https://code.org/educate/curriculum/apcsa). Review our
-          [guidance documents](https://docs.google.com/document/d/1DhvzoNElJcfGYLrp5sVnnqp0ShvsePUpp3JK7ihjFGM/edit)
-          to see whether your course implementation plans meet our program guidelines."
+          Placement CSA (in Java), visit this
```

Comment:
should there be a space in CS A? I have no idea what the preferred way to write it is, just asking!

Reply:
removing the space was explicitly put in the spec, so I think it's the preferred way!